eduNEXT/eox-nelp
# feat: implement multiple configurations

## Description
This implements the [multiple configurations](https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations)  webpack structure to  allow plugin modifications per plugin


## Testing instructions

1. Run npm run build
2. Go to the stats and feedback page and verify
3. Run npm run dev
4.  Go to the stats and feedback page and verify



## Checklist for Merge
